acceptance-tests/build.gradle

```diff
@@ -22,6 +22,12 @@ dependencies {
                        "org.seleniumhq.selenium:selenium-java:4.1.3",
                        "org.junit.jupiter:junit-jupiter-api:${dependencyVersions.junit_version}"
 
+    testImplementation 'io.cucumber:cucumber-java:6.10.4'
+    testImplementation 'io.cucumber:cucumber-junit:6.10.4'
```

Comment:
We could use a variable for this dependency version. See `junit_version`

Reply:
Thanks @JHjava, it looks like those were a leftover of early experiments when I was trying to make it work. I added variables for axe version and json version.